src/MoBi.Engine/Sbml/MoleculeInformation.cs

```diff
@@ -21,7 +21,7 @@ public class MoleculeInformation
         /// </summary>
         public bool IsMultipleTimesInOneCompartment()
         {
-            return _container.Select(container => container.Name).Any(name => _container.Any(con => name == con.Name));
+            return _container.Any(container=> _container.Any(con => container != con && container.Name == con.Name));
```

Comment:
I have no idea what this does? container, _container, con ... 
 What are you checking here? I would really suggest to use x for the linq variable in general so that one can see right away what is the variable and what is fixed. Two queries like this imbricated is very hard to read.